accounts/abi/abi.go

```diff
@@ -274,12 +274,21 @@ func (abi ABI) Unpack(v interface{}, name string, output []byte) error {
 		// struct will match named return values to the struct's field
 		// names
 		case reflect.Struct:
+			idx := 0
```

Comment:
Sorry, I'm not sure what you mean. I declare idx as an alternative to i, as the index into output must be separate from the index into the number of outputs.